mm2src/coins/lp_coins.rs

```diff
@@ -2295,6 +2295,8 @@ pub enum ValidatorsInfoDetails {
 pub struct SignatureRequest {
     coin: String,
     message: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Why `skip_serializing_if` is needed in the request? I guess we need to remove deriving `Serialize` as well

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2432/commits/b4531266eae378da1ea48d0a6aacad0efc3bc67b